src/store/mutations.js

```diff
@@ -443,6 +444,10 @@ function SOCKET_ONCLOSE(state) {
   }
 }
 
+function SOCKET_ONERROR() {
+  // Socket error...
```

Comment:
Do we not have the code that should go here?

Reply:
Correct. This is just to stop console errors.